src/clients/cli.rs

```diff
@@ -195,6 +223,53 @@ impl Docker for Cli {
     }
 }
 
+impl Drop for Cli {
```

Comment:
Are you testing that dropping the cli stops the container?

Reply:
Dropping the client does not stop the container, dropping the container stops the container :)

Rust lifetimes guarantee that the containers are dropped before the client.